tap_listrak/__init__.py

```diff
@@ -38,23 +37,27 @@ def discover(ctx):
         for field_name in schema_dict['properties'].keys():
             mdata = metadata.write(mdata, ('properties', field_name), 'inclusion', 'automatic')
 
+        md_list = metadata.to_list(mdata)
+        for md in md_list:
+            if md["metadata"].get("inclusion") != "unsupported":
+                md["metadata"]["selected"] = True
+
         catalog.streams.append(CatalogEntry(
             stream=tap_stream_id,
             tap_stream_id=tap_stream_id,
             key_properties=schemas.PK_FIELDS[tap_stream_id],
             schema=schema,
-            metadata = metadata.to_list(mdata)
+            metadata=md_list
         ))
```

Comment:
This by default seelecting all the fields, this is counter productive to feild selection feature. UI selection will populate the selected field in the catalog, so no need to handle this in the code.